tests/Behat/FeatureContext.php

```diff
@@ -72,6 +104,29 @@ public function findDateFields($field) {
     return $field_selectors;
   }
 
+  /**
+   * Finds a daterange field.
+   *
+   * @param string $field
+   *   The field name.
+   *
+   * @return \Behat\Mink\Element\NodeElement[]
+   *   The elements found.
+   *
+   * @throws \Exception
+   *   Thrown when the field was not found.
+   */
+  public function findDateRangeFields($field) {
+    $field_selectors = $this->getSession()->getPage()->findAll('css', '.field--widget-daterange-default');
+    $field_selectors = array_filter($field_selectors, function ($field_selector) use ($field) {
+      return $field_selector->has('named', ['content', $field]);
+    });
+    if (empty($field_selectors)) {
+      throw new \Exception("Date range field {$field} was not found.");
+    }
+    return $field_selectors;
+  }
+
```

Comment:
We should move both of these steps in reusable context, so that we can re-use them in, for example, EWCMS Behat tests. We can call the context `DateFieldContext`.